godtools/Views/TractElements/TractElementFunctions.swift

```diff
@@ -84,4 +85,17 @@ extension BaseTractElement {
         return (elementName!, namespace, kind, properties, children)
     }
     
+    func elementListener() -> String {
+        return ""
+    }
+    
+    func sendMessageToElement(tag: String) {
```

Comment:
For the record, what I told you on Hipchat:

I created a dictionary for setting all the views that have a listener (lets call it dictionaryX), so, if an interaction is generated (like hitting a button), the button will dispatch a message with the values associated, and those values are going to be searched in the dictionaryX, and it will return the elements (views) with those listeners and will process the message, thats why I didn't see the need of using notifications for that (my only concern was about sending parameters to the view, but I already have an idea that I can develop on the top of that); were you talking about that or something else that I'm missing?